app/controllers/stories_controller.rb

```diff
@@ -293,17 +297,17 @@ def assign_user_stories
     @stories = ArticleDecorator.decorate_collection(@user.articles.published.
       limited_column_select.
       where.not(id: @pinned_stories.pluck(:id)).
-      order("published_at DESC").page(@page).per(user_signed_in? ? 2 : 20))
+      order("published_at DESC").page(@page).per(user_signed_in? ? 2 : SIGNED_OUT_RECORD_COUNT))
   end
 
   def stories_by_timeframe
     if %w[week month year infinity].include?(params[:timeframe])
       @stories.where("published_at > ?", Timeframer.new(params[:timeframe]).datetime).
         order("positive_reactions_count DESC")
     elsif params[:timeframe] == "latest"
-      @stories.where("score > ?", -40).order("published_at DESC")
+      @stories.where("score > ?", -20).order("published_at DESC")
     else
-      @stories.order("hotness_score DESC")
+      @stories.order("hotness_score DESC").where("score > 2")
```

Comment:
Ensure things don't escape the "latest" tab unless there is a positive score. Anything thumbsdown'd will likely not stick around for crawling.